Segments/Blackburrow.mapproj

```diff
@@ -91494,7 +91559,7 @@ if (spokeRecently(source))
 {
     Name = "Potion Master Nook",
     Alignment = Alignment.Chaotic,
-    Body = 388,
+    Body = 387,
```

Comment:
Can we confirm that this is compiled and can be used?

Reply:
I would assume yes.. while they dont match with the Bestiary. 883-388 work but dont match the website. 388 is suppose to be a shaman dogman. and 387 is a variant shaman dogman.. but 388 is a wolf with a sword.